### PR TITLE
[NFC/cleanup] Remove more entries of variables.

### PR DESCRIPTION
**Pull request by rdzhabarov**
*Description*:
Removing references to variables when appropriate.

*Testing*:
unit tests

*Documentation*:
N/A

Related to https://github.com/pytorch/glow/issues/1334

### CODE REVIEW

**Comment by yinghai** [comment]
Noob question: what's the reasoning of changing `variable` to `input`?

**Comment by rdzhabarov** [comment]
That's a long discussion. See https://github.com/pytorch/glow/issues/1334 for details.
This PR is cleaning up what was left after migration.

File: lib/Optimizer/Partition.cpp [hunk 1]
```diff
@@ -96,7 +96,7 @@ NodeFunctionMap selectBasicBlockPartitions(Function *F) {
 
     // If node has only one input, and that input has only one output, place it
     // in the same partition.
-    auto *in = singleNonVariableInput(node);
+    auto *in = singleNonStorageInput(node);
```
Comment (jfix71) [inline lib/Optimizer/Partition.cpp]: Just to be sure -- was the intention here for any Storage node? Or was it really intended more for what are now Placeholder inputs? @bertmaher

**Comment by rdzhabarov** [comment]
I like `updateInputPlaceholders`, more explicit to what's going on.

**Pull request merged**